pybossa/auth/project.py

```diff
@@ -66,8 +66,6 @@ def _update(self, user, project):
         return self.only_admin_or_subadminowner(user, project)
 
     def _delete(self, user, project):
-        if self.result_repo.get_by(project_id=project.id):
```

Comment:
Why was the code blocking deletion of projects with task results?
It looks like the DB schema is set up to cascade deletes from project to all of its references.
Will removing this code cause any problems?